modules/videoio/src/cap_ffmpeg_impl.hpp

```diff
@@ -1408,7 +1408,7 @@ bool CvCapture_FFMPEG::retrieveFrame(int, unsigned char** data, int* step, int*
             img_convert_ctx,
             picture->data,
             picture->linesize,
-            0, video_st->codec->coded_height,
+            0, video_st->codec->height,
```

Comment:
Again, please use `picture->height` here.

According to ffmpeg sample code: https://github.com/FFmpeg/FFmpeg/blob/n4.4.2/fftools/ffplay.c#L925-L926

Reply:
You are right, we use picture->data and picture->linesize so it make sense.
Done (build and run OK)